lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriApiService.java

```diff
@@ -207,7 +205,7 @@ public VerifiableCredentialResponse fetchVerifiableCredential(
                 var verifiableCredentialResponse =
                         VerifiableCredentialResponse.builder()
                                 .verifiableCredentials(
-                                        Collections.singletonList(response.getContent()))
+                                        Collections.singletonList(response.getContent().trim()))
```

Comment:
Are you sure this is correct? It doesn't necessarily do any harm, but it looks a bit suspicious.